app/src/androidTest/java/org/mozilla/focus/activity/screenshots/ScreenGrabTest.java

```diff
@@ -482,16 +476,21 @@ private void takeScreenshotOfErrorPages(UiDevice device) throws UiObjectNotFound
 
             assertTrue(TestHelper.webView.waitForExists(waitingTime));
 
-            onWebView()
-                    .withTimeout(loadingWaitingTime, TimeUnit.MILLISECONDS)
-                    .withElement(findElement(Locator.ID, "errorTitle"))
-                    .perform(webClick());
-
-            onWebView()
-                    .withTimeout(loadingWaitingTime, TimeUnit.MILLISECONDS)
-                    .withElement(findElement(Locator.ID, "errorTryAgain"))
-                    .perform(webScrollIntoView());
-
+            // Android O has an issue with using Locator.ID
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+                UiObject tryAgainBtn = device.findObject(new UiSelector()
+                        .descriptionContains(context.getString(R.string.errorpage_refresh))
+                        .clickable(true));
+                assertTrue(tryAgainBtn.waitForExists(waitingTime));
+            } else {
+                onWebView()
```

Comment:
\[Checkstyle\] ERROR: '\+' is not preceded with whitespace\.

Reply:
hmm, lint bug perhaps?